pkg/controller/provider/controller.go

```diff
@@ -172,6 +172,15 @@ func (r Reconciler) Reconcile(request reconcile.Request) (result reconcile.Resul
 		r.Log.V(2).Info("Conditions.", "all", provider.Status.Conditions)
 	}()
 
+	defer func() {
+		// Stop reconciliation when auth fails
+		if provider.Status.HasCondition(ConnectionAuthFailed) {
+			result.RequeueAfter = 0
+			err = nil
+			return
```

Comment:
how do we get out of this "loop" then, does the reconciliation continue when the url or credentials change?

Reply:
ack